### PR TITLE
Add python3.6 python3.7 support to manylinux Dockerfile

### PR DESCRIPTION
**Pull request by velconia**
According to this [issue](https://github.com/PaddlePaddle/Paddle/issues/12040), we should support python 3.6 and python 3.7 in manylinux1 Dockerfile

### CODE REVIEW

File: tools/manylinux1/Dockerfile.x64 [hunk 1]
```diff
@@ -36,17 +36,21 @@ RUN cd /opt && wget -q --no-check-certificate https://github.com/google/protobuf
     tar xzf protobuf-cpp-3.1.0.tar.gz && \
     cd protobuf-3.1.0 && ./configure && make -j4 && make install && cd .. && rm -f protobuf-cpp-3.1.0.tar.gz
 
-RUN wget -O /root/requirements.txt https://raw.githubusercontent.com/PaddlePaddle/Paddle/develop/python/requirements.txt
+RUN wget https://raw.githubusercontent.com/PaddlePaddle/Paddle/develop/python/requirements.txt -O /root/requirements.txt
```
Comment (typhoonzero) [inline tools/manylinux1/Dockerfile.x64:39]: what is this change for?

File: tools/manylinux1/build_scripts/build_utils.sh [hunk 2]
```diff
@@ -52,9 +52,15 @@ function do_cpython_build {
 
     # NOTE --enable-shared for generating libpython shared library needed for
     # linking of some of the nupic.core test executables.
-    CFLAGS="-Wformat" ./configure --prefix=${prefix} --enable-shared $unicode_flags > /dev/null
-    make -j2 > /dev/null
-    make install > /dev/null
+    if [ $(lex_pyver $py_ver) -ge $(lex_pyver 3.7) ]; then
+        CFLAGS="-Wformat" ./configure --prefix=${prefix} --with-openssl=/usr/local/ssl --enable-shared $unicode_flags > /dev/null
```
Comment (typhoonzero) [inline tools/manylinux1/build_scripts/build_utils.sh:58]: better to add notes about why need to deal with version `3.7`

**Review by typhoonzero — DISMISSED** [review]
LGTM with minor comments.

File: tools/manylinux1/build_scripts/build_utils.sh (hunk 2 above)
Comment (velconia) [reply tools/manylinux1/build_scripts/build_utils.sh:58]: Thanks for reviewing, I added it now

File: tools/manylinux1/Dockerfile.x64 (hunk 1 above)
Comment (velconia) [reply tools/manylinux1/Dockerfile.x64:39]: Well this is a little trick for removing the requirements' cache

**Review by junjun315 — APPROVED** [review]
ok

**Pull request merged**